ports/rp2/boards/WEACTSTUDIO/weactstudio_16mb.h

```diff
@@ -0,0 +1,73 @@
+// A pico-sdk board definition is required since the Wavgat boards are not officially
```

Comment:
Can we make `weactstudion_common.h` and have them all share that except for setting `PICO_FLASH_SIZE_BYTES`.

Reply:
Yes, though I just called the common file `weactstudio.h`.